tests/unittests/tests-libcose/tests-libcose.c

```diff
@@ -0,0 +1,222 @@
+﻿/*
+ * Copyright (C) 2018 Freie Universität Berlin
+ * Copyright (C) 2018 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+ /**
+  * @ingroup    tests
+  * @{
+  *
+  * @file
+  * @brief      Unit tests for pkg libcose
+  *
+  * @author     Koen Zandberg <koen@bergzand.net>
+  */
+
+#define MAX_NUMBER_BLOCKS 128
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include <tweetnacl.h>
```

Comment:
Why it's included with `<>`? Since it's not a system header I'd say it needs to be with `""`.

Reply:
Fixed